pages/common/dlv.md

```diff
@@ -0,0 +1,24 @@
+# dlv
+
+> Debugger for the Go programming language.
+> More information: <https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv.md/>.
+
+- Compile and begin debugging main package in current directory, or the package specified:
+
+`dlv debug`
+
+- Compile test binary and begin debugging program:
+
+`dlv test`
+
+- Connect to a headless debug server:
+
+`dlv connect {{id_address}}`
+
+- Attach to running process and begin debugging:
+
+`div attach {{pid}}`
+
+- Compile and begin tracing program:
+
+`dlv trace {{package}} --regexp {{flags}}`
```

Comment:
I'm not suer I understand the `flags` here. Would this make more sense?
```suggestion
`dlv trace {{package}} --regexp '{{regex_pattern}}'`
```

Also, generally  I recommend enclosing regular expressions in single quotes `'`.